build.gradle

```diff
@@ -61,6 +60,59 @@ sonar {
     }
 }
 
+def getFixAndFullVersion() {
+    def currentVersion = currentVersion.split("\\.")
+    def lastVersionNumber = currentVersion[currentVersion.length - 1]
+
+    return [lastVersionNumber, currentVersion]
+}
+
+def setVersionProperty(String newVersion) {
+    ant.propertyfile(file: "gradle.properties") {
```

Comment:
Should we really use Apache Ant here? This adds another external dependency (Apache Ant) to the already complicated toolchain and dependency list, cannot we do this with basic Gradle logic instead?

Reply:
Apache ant is already being used by Gradle and is not a new dependency. This is the only suitable solution that updates the version and does not reformat the text each time.

Reply:
Are you sure? I don't think we currently use Ant in any of our Gradle builds/scripts? Or am I overlooking something?

Reply:
Have a look https://docs.gradle.org/current/userguide/ant.html

Reply:
I know that using Ant in Gradle builds is possible, however I don't think we do use it in any of our Gradle builds so far? Hence my question if we really should start doing it now for something "simple" like replacing a property value in a property file? Could not that be done without Ant too?

Reply:
> Apache ant is already being used by Gradle and is not a new dependency. This is the only suitable solution that updates the version and does not reformat the text each time.

Kindly refer to the above reply, the other options that I had looked at were reformatting the whole file which will always result in a diff that is unnecessary.